cli/io/fio.py

```diff
@@ -0,0 +1,114 @@
+import itertools
+
+from cli import Cli
+from cli.utilities.packages import Package
+
+
+class FioError(Exception):
+    pass
+
+
+class Fio(Cli):
+    def __init__(self, client):
+        super(Fio, self).__init__(client)
+        self.client = client
+
+    def install_fio(self):
+        """
+        Install Fio
+        """
+        try:
+            # Install Fio packages
+            Package(self.client).install("fio", nogpgcheck=True)
+        except Exception:
+            raise FioError("Fio installation failed")
+
+    def create_config(
+        self, ioengine, mount_dir, workloads, sizes, iodepth_values, numjobs
+    ):
+        """
+        Generates all the combimations of workloads sizes, iodepth as per the fio_config
+        Sample fio_config:
+        fio_config:
```

Comment:
I assume these are test suite configs. Remove this and add description for parameters passed to this method